scrapers/covid-19/govScrapers/getCanada.js

```diff
@@ -1,39 +1,31 @@
 const axios = require('axios');
-const cheerio = require('cheerio');
 const logger = require('../../../utils/logger');
-
-const columns = ['province', 'cases', 'deaths'];
+const csvUtils = require('../../../utils/csvUtils');
+const moment = require('moment');
 
 /**
- * Return object reflecting a row of data from Canadian government site
- * @param 	{number} 	_ 		Index getting passed when using .map()
- * @param 	{Object} 	row		The row to extract data from
- * @returns {Object}				Data for canadian province with entries for each column in @constant columns
+ * Return array of provinces that match today's date (initial csv is historical)
+ * @param 	{Object} 	csv		The row to extract data from
+ * @returns {Array}				Data for canadian province
  */
-const mapRows = (_, row) => {
-	const province = { updated: Date.now() };
-	cheerio(row).children('td').each((index, cell) => {
-		cell = cheerio.load(cell);
-		switch (index) {
-			case 0: {
-				province[columns[index]] = cell.text() === 'Canada' ? 'Total' : cell.text();
-				break;
-			}
-			default: {
-				province[columns[index]] = parseInt(cell.text().replace(/,/g, '')) || null;
-			}
-		}
-	});
-	return province;
+const filterByDate = (csv) => {
+	const date = moment().format('DD-MM-YYYY');
```

Comment:
The entire reason tests are failing, and no data is returned is because you are checking for today's date in the CSV. You need to check YESTERDAY'S date, since the CSV is only up to date as of the day before. Then, tests will work.